src/core/lib/iomgr/ev_epollex_linux.cc

```diff
@@ -767,6 +768,10 @@ static int poll_deadline_to_millis_timeout(grpc_millis millis) {
 }
 
 static void fd_become_readable(grpc_fd* fd, grpc_pollset* notifier) {
+  if (grpc_latency_estimator_trace.enabled()) {
+    gpr_log(GPR_INFO, "latency_estimator:server_inbound_reading:start");
```

Comment:
don't we need to disambiguate if we are a client or server process? Else how will this interact correctly with the post processor?

Reply:
I could modify the post processor to have a client and server mode. So when it's in client mode, it throws out all the server-related log statements. 
Maybe the right thing to do at this point is to define where we want to put the start and end markers for each leg of an rpc? And perhaps that will solve the disambiguation problem just by nature of where the markers are.

Reply:
@ncteisen Talked with Abhishek, and he said it's ok to leave the log statements where they are now, as long as it's consistent between runs. I think modifying the post processor to have the two separate modes should work as a quick solution.

Reply:
That solution sounds good to me!